packages/lab/src/radio-button/RadioButton.css

```diff
@@ -1,57 +1,35 @@
-/* Styles applied to root component  */
-.salt-density-medium {
-  --radioButton-label-marginTop: 6px;
-  --radioButton-row-marginRight: 24px;
-}
-
-.salt-density-high {
-  --radioButton-label-marginTop: 3px;
-  --radioButton-row-marginRight: 12px;
-}
-
-.salt-density-low {
-  --radioButton-label-marginTop: 9px;
-  --radioButton-row-marginRight: 36px;
-}
-
-.salt-density-touch {
-  --radioButton-label-marginTop: 12px;
-  --radioButton-row-marginRight: 48px;
-}
-
+/* Styles applied to RadioButton container */
 .saltRadioButton {
+  display: flex;
+  gap: calc(var(--salt-size-unit) * 0.75);
+  position: relative;
+  user-select: none;
```

Comment:
Just curious, was this to solve an issue?

Reply:
Not sure, I assumed it was intended. I've pulled it over from RadioButtonBase. You think I should remove it?